images/hub/Dockerfile

```diff
@@ -36,6 +36,9 @@ RUN pip3 install --no-cache-dir \
          -r /tmp/requirements.txt
 
 ADD jupyterhub_config.py /srv/jupyterhub_config.py
+
+# FIXME: Just adding this to current dir doesn't seem to work
```

Comment:
Current directory is only on PATH when doing `python -i` or other such interactive things. CWD is not normally on PYTHONPATH when launching programs. Putting it here would work if we had:

```python
import os, sys
sys.path.insert(0, os.path.dirname(__file__))
```

in config.py.

Reply:
Hmm, but it worked when I did `jupyterhub -f jupyterhub_config.py` locally too...

Reply:
I'll re-test in a bit.

Reply:
never managed to re-test, but current code is good enough I guess :)